### ISSUE TITLE
node_modules doesn't need to be in repository

### ISSUE THREAD

**Issue by cllofty** [scott-lofty/want-to-play-bot#15]
The `node_modules` folder is generated when installing libraries from the source controlled `package.json` folder. I consider this folder "compiled" code and doesn't need to be in source control.

Also, the folder is `97M` in size:

```
>du node_modules -sh
97M	node_modules
```

**Comment by scott-lofty** [comment]
Yes. Can you remove it and add it to get ignore. I discovered the last
night while trying to run it on my desktop
